socorro/external/es/crashstorage.py

```diff
@@ -303,6 +303,10 @@ class ESCrashStorageRedactedJsonDump(ESCrashStorageRedactedSave):
         )
     )
 
+    def is_mutator(self):
+        # This crash storage mutates the crash, so we mark it as such.
+        return True
```

Comment:
This is the class that we use that mutates the data. I didn't go and check to see if other classes that we don't use mutate the data. We can write up another bug for that.